packager/hls/base/master_playlist.cc

```diff
@@ -55,6 +55,14 @@ struct Variant {
   uint64_t avg_audio_bitrate = 0;
 };
 
+// This structure is used to store the playlist and its tags.
+struct MediaTagslist {
```

Comment:
Something about the name `MediaTagslist` is too visually close to `MediaPlaylist` to the point where I had to read the code closely several times to make sure I was reading it correctly. Perhaps we can find a better name.